builds/checkin.yaml

```diff
@@ -19,6 +19,6 @@ jobs:
       - script: scripts/fetch.sh dunfell
         displayName: Fetch
         env:
-          METARUST_REV: 'e4d25b98083bcecb94df6ee189a165d63ede7f3d'
+          METARUST_REV: '7ff669d8cedd83a2d3efb73073a63b0a7efffddc'
```

Comment:
I think it _may_ be a good idea to pin all of the other repo revision as well for the sake of reproducibility? 
Namely `POKY_REV`, `METAOE_REV`, and `METAVIRT_REV`.

Reply:
Good comment, I'll have to think about this - I don't know how much these layers change or get updated to have this be an issue.  I would personally prefer to ensure we are compatible with the latest revisions.